packit_service/worker/jobs.py

```diff
@@ -142,7 +145,10 @@ def get_config_for_handler_kls(
 
     matching_job_types = MAP_HANDLER_TO_JOB_TYPES[handler_kls]
     for job in jobs_that_can_be_triggered:
-        if job.type in matching_job_types and job not in matching_jobs:
+        if (
+            any(are_job_types_same(job.type, type) for type in matching_job_types)
```

Comment:
this is starting to be more and more complicated: if you still have this particular change in your mind, could you please comment in the code here what's happening?

Reply:
Still same story about `build` x `copr_build`...;( I'll make a note here.

Reply:
> this is starting to be more and more complicated

Sadly, yes. Those aliases and requires don't help us much. I hope we can refactor this once we clear the enums a bit.

Reply:
Comment added.